var/spack/repos/builtin/packages/upcxx/package.py

```diff
@@ -16,7 +16,7 @@ def is_CrayXC():
 
 
 def is_CrayEX():
-    return (spack.platforms.host().name == "cray") and (
+    return (spack.platforms.host().name == "linux" or spack.platforms.host().name == "cray") and (
```

Comment:
FWIW. You could simplify this as:

```suggestion
    return (spack.platforms.host().name in ["cray", "linux"]) and (
```

Reply:
Hi @tldahlgren, thanks for the suggestion - resolved in 650f15dba5